src/components/lesson-item/LessonItem.component.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC, useEffect } from 'react';
+import clsx from 'clsx';
+import { FaLock, FaPlayCircle, FaUnlink } from 'react-icons/fa';
+
+import { VideoLesson } from '../../types/types';
+
+import './LessonItem.scss';
+
+interface LessonItemProps {
+  index: number;
+  title: VideoLesson['title'];
+  status: VideoLesson['status'];
+  link: VideoLesson['link'];
+  order: number;
+  duration: VideoLesson['duration'];
+  handleChangeLessonData: (videoSrc: string, imagePreviewLink: string) => void;
+  activeLessonVideoLink: string;
+  previewImageLink: string;
+}
+
+export const LessonItem: FC<LessonItemProps> = ({
+  index,
+  status,
+  title,
+  link,
+  order,
+  duration,
+  handleChangeLessonData,
+  activeLessonVideoLink,
+  previewImageLink,
+}) => {
+  const lockedStatus = status === 'locked';
+  const noVideoLink = !link || link.length === 0;
+  const isLessonActive = link && activeLessonVideoLink === link;
+  const activeLessonPreviewImage = `${previewImageLink}/lesson-${order}.webp`;
+
+  const selectLessonToView = () => {
+    if (!lockedStatus) {
+      handleChangeLessonData(link, activeLessonPreviewImage);
+      window.scrollTo({ top: 0, behavior: 'smooth' });
+    }
+  };
+  const itemStyles = clsx(`lesson__item`, {
+    lesson__item_active: isLessonActive,
+    lesson__item_locked: lockedStatus || noVideoLink,
+  });
+
+  useEffect(() => {
+    if (order === 1) {
+      console.log(link);
+
+      handleChangeLessonData(link, activeLessonPreviewImage);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div className={itemStyles}>
```

Comment:
why you used camel Case now ?

Reply:
``itemStyles`` in camelCase because it is a variable that comes from the code above (was used ``clsx`` lib) and not directly from .scss styles

Reply:
Here it is: 
```
const itemStyles = clsx(`lesson__item`, {
    lesson__item_active: isLessonActive,
    lesson__item_locked: lockedStatus || noVideoLink,
  });
```